.travis.yml

```diff
@@ -12,15 +12,29 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-language: java
+matrix:
+  include:
+    - language: go
 
-services:
-  - docker
-  
-install: true
+      env:
+        - GO111MODULE=on
 
-script: docker run -v "$HOME/.m2":/root/.m2 --rm -v "$(pwd)":/usr/src/mymaven -w /usr/src/mymaven a5anka/wso2-ei-maven mvn clean install -Dsurefire.useSystemClassLoader=false -fae -B -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
+      before_install:
+        - cd cmd/
+        - go mod download
```

Comment:
If we cache the module download, will it speedup the build?

Reply:
Yes, I have updated the code in dda2e3a